native/c/zowex.cpp

```diff
@@ -186,6 +186,7 @@ int main(int argc, char *argv[])
 
   // Common data set options that are reused
   auto encoding_option = make_aliases("--encoding", "--ec");
+  auto source_encoding_option = make_aliases("--local-encoding", "--sec");
```

Comment:
```suggestion
  auto source_encoding_option = make_aliases("--local-encoding", "--lec");
```

Reply:
Or Maybe “source” and “target” encoding are clearer terms?